src/scss/overrides/_type.scss

```diff
@@ -76,3 +86,24 @@ a {
     font-weight: 100;
   }
 }
+
+.jumbotron {
+  h1, h2, h3, h4, h5, h6 {
+    color: $white;
+    margin: 0;
+  }
+  h4 {
+    opacity: 0.6;
+  }
+}
+
+ul {
+  padding-left: 14px;
+  line-height: 32px;
```

Comment:
This probably should be in ems or relative to the font size with a unitless value.